org.eclipse.buildship.core/schema/executionparticipants.exsd

```diff
@@ -19,7 +19,7 @@
       </annotation>
       <complexType>
          <choice minOccurs="1" maxOccurs="unbounded">
-            <element ref="plugin"/>
+            <element ref="buildparticipant"/>
```

Comment:
There is existing extension point that is expecting the 'plugin' element.  See here:  https://github.com/eclipse/buildship/blob/master/org.eclipse.buildship.ui/plugin.xml#L468-L470

So for this feature you need to add a new element for 'class' but don't remove existing element.

Reply:
ok, I will restore plugin part.